src/main/java/com/google/devtools/build/lib/buildtool/ExecutionTool.java

```diff
@@ -984,7 +984,7 @@ private static AbruptExitException createExitException(
     return new AbruptExitException(
         DetailedExitCode.of(
             FailureDetail.newBuilder()
-                .setMessage(message)
+                .setMessage(String.format("%s: %s", message, e.getMessage()))
```

Comment:
Could you rename the `message` parameter to something like `messagePrefix` to make it clear that it's prepended to the exception's message?

Reply:
Great suggestion. Done!